.github/home.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <link rel="stylesheet" href="home.css">
+</head>
+<body>
+    <header>
+        <div class="head">
+            <a href="home.html"><img src="img/logo.svg"></a>
```

Comment:
```suggestion
            <a href="home.html"><img src="img/logo.svg" /></a>
```
이미지 태그는 self-closing이 가능합니다.
지금은 넣어주지 않아도 동작을 하겠지만, 차후 react 개발을 할 때 예상치 못한 오류를 발생시킬 수 있습니다.
self-closing 할 수 있는 요소는 잊지 말고 입력해주세요